### ISSUE TITLE
Orange tag has insufficient contrast

### ISSUE THREAD

**Issue by aardrian** [tc39/tc39.github.io#139]
The orange tag for specification has insufficient contrast with its white text (see [WCAG 2.1 SC 1.4.3](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141#contrast-minimum) and [G18: Technique for meeting 1.4.3](https://www.w3.org/TR/WCAG20-TECHS/G18.html)). The text is white (#fff) and the background is orange (#f90), producing a contrast ratio of 2.1:1, well below the 4.5:1 minimum.

![2019-03-19 (2)](https://user-images.githubusercontent.com/1376607/54600790-ee0a5780-4a13-11e9-9d0d-791692cfbd12.png)

Orange and white generally have poor contrast, so you will end up with an almost brown color if you try to darken it to meet 4.5:1.

**Comment by littledan** [comment]
@aardrian Thanks for raising this issue. Would you be interested in making a PR to fix it?

**Comment by aardrian** [comment]
Sure. Do you have a color palette already defined or some other parameters? Or can I just make a choice/suggestion?

**Comment by littledan** [comment]
Suggestions would be welcome.

**Comment by aardrian** [comment]
Fixed with https://github.com/tc39/tc39.github.io/pull/140.
